core/src/test/java/org/elasticsearch/cluster/metadata/MetaDataMappingServiceTests.java

```diff
@@ -109,7 +109,7 @@ public void testClusterStateIsNotChangedWithIdenticalMappings() throws Exception
         final MetaDataMappingService mappingService = getInstanceFromNode(MetaDataMappingService.class);
         final ClusterService clusterService = getInstanceFromNode(ClusterService.class);
         final PutMappingClusterStateUpdateRequest request = new PutMappingClusterStateUpdateRequest().type("type");
-        request.source("{ \"properties\" { \"field\": { \"type\": \"string\" }}}");
+        request.source("{ \"properties\" { \"field\": { \"type\": \"text\" }}}");
```

Comment:
@jpountz I guess this change is ok? Any idea why nothing is failing on master because of this?

Reply:
This doesn't look good. This test must be broken is some way.

Reply:
Would you prefer to revert changes in this test and have a look yourself at it?

Reply:
OK I was lacking context actually. The  use of `string` potentially hid a bug, switching to `text` fixes it. +1 to change it.